app/assets/stylesheets/navigation.scss

```diff
@@ -1,76 +1,60 @@
-$side-nav-width: 240px;
+$nav-avatar-size: 34px;
 
-nav.top-nav {
-  background-color: #FFF;
+nav {
+  $bg-color: #FFF;
+  $font-color: #456293;
 
-  a.page-title {
-    color: $secondary-color;
-    font-weight: 300;
-
-    i { font-size: 1.2rem; }
-  }
-}
-
-.side-nav.fixed {
-  $bg-color: color("grey", "lighten-3");
-  $link-color: $secondary-color;
-  $line-height: 64px;
-
-  width: $side-nav-width;
   background-color: $bg-color;
 
-  li {
-    &:hover, &.active { background-color: darken($bg-color, 5%); }
-  }
-
-  li.brand-logo {
-    a {
-      font-size: 1.15rem;
-      font-weight: 700;
-
-      i { margin-right: 5px; }
+  .button-collapse {
+    i {
+      color: $font-color;
+      font-size: 1.8rem;
+      margin-right: 1rem;
     }
   }
 
-  li.user-area {
-    img.avatar {
-      margin-right: 10px;
-      vertical-align: middle;
-      width: 34px;
-      height: 34px;
-    }
+  .brand-logo {
+    color: $font-color;
+    font-size: 1.2rem;
+    font-weight: 700;
   }
 
-  a {
-    color: $link-color;
-    font-weight: 700;
-    i { line-height: $line-height; }
+  ul.right {
+    i {
+      color: lighten($secondary-color, 25%);
+    }
   }
 }
 
-@media #{$large-and-up} {
-  header, main {
-    padding-left: $side-nav-width;
+li.avatar {
+  img {
+    vertical-align: middle;
+    width: $nav-avatar-size;
+    height: $nav-avatar-size;
+    border-radius: 0.3rem;
   }
-  .container {
-    width: 90%;
+
+  i {
+    margin-left: 3px;
   }
 }
 
-@media #{$medium-and-down} {
-  nav .nav-wrapper {
-    text-align: center;
+.side-nav {
+  $bg-color: color("grey", "lighten-3");
+  $font-color: $primary-font-color;
+
+  $line-height: 64px;
+
+  background-color: $bg-color;
+
+  a {
+    color: $font-color;
   }
-}
 
-a.button-collapse.top-nav {
-  position: absolute;
-  text-align: center;
-  left: 7.5%;
-  top: 0;
-  float: none;
-  margin-left: 0.3rem;
-  color: black;
-  font-size: 32px;
-  z-index: 2;
+  i { line-height: $line-height; }
+
+  li.avatar {
+    span { left-margin: 5px; }
```

Comment:
Unknown property left-margin<br>Selector should have depth of applicability no greater than 2, but was 3